src/api/style.js

```diff
@@ -8,13 +8,13 @@ import Expression from '../core/style/expressions/expression';
 import CartoValidationError from './error-handling/carto-validation-error';
 
 
-const DEFAULT_RESOLUTION = 1;
-const DEFAULT_COLOR_EXPRESSION = s.rgba(0, 1, 0, 0.5);
-const DEFAULT_WIDTH_EXPRESSION = s.float(5);
-const DEFAULT_STROKE_COLOR_EXPRESSION = s.rgba(0, 1, 0, 0.5);
-const DEFAULT_STROKE_WIDTH_EXPRESSION = s.float(0);
-const DEFAULT_ORDER_EXPRESSION = s.noOrder();
-const DEFAULT_FILTER_EXPRESSION = s.floatConstant(1);
+const DEFAULT_RESOLUTION = () => 1;
```

Comment:
Why are this constants turned to functions?

Reply:
"However, there was another problem, expressions with multiple parents are not supported, but style.js was sharing expressions between styles, this caused some WebGL warnings and it was going to explode sooner or later, so I fixed it." we need to create a new expression each time we need them